### DISCUSSION TITLE
Version number only updates after warm start or power cycle

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1312] [Fixed Issues]
Was running rAudio 20230522

After applying today's update, the version number remains the same (20230522).

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by Sagnasty** [comment]
My mistake - was looking at the wrong system prior to updating

**Comment by Sagnasty** [comment]
While testing the rAudio 20230528 update, I noticed that after successfully applying the update, the rAudio version number in the Systems page remained unchanged.

Only a warm start or power cycle will update the version number to the correct value.

I have updated the category and title to better describe the issue.

**Reply by ceejayemm** [reply]
Same here having updated to 20230528 the System page still shows 20230525.

Chris

**Comment by Sagnasty** [comment]
Problem still exists with rAudio 20230603

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by Sagnasty** [comment]
Fixed in 20230609 - Thanks Rern

Jim
